src/wireviz/__init__.py

```diff
@@ -1,3 +1,7 @@
 # Please don't import anything in this file to avoid issues when it is imported in setup.py
 
 __version__ = '0.1.1'
+
+APP_NAME = 'WireViz'
+CMD_NAME = APP_NAME.lower()
+URL = 'https://github.com/formatc1702/' + APP_NAME
```

Comment:
`URL = 'https://github.com/formatc1702/WireViz'` is fine, building up a URL using a constant feels like overkill.

Reply:
Maybe rename to `APP_URL` to keep it consistent with `APP_NAME`